matchms/filtering/add_precursor_mz.py

```diff
@@ -1,16 +1,32 @@
 import logging
 from matchms.utils import get_first_common_element
-from ..typing import SpectrumType
 
 
 logger = logging.getLogger("matchms")
 
 
-_accepted_keys = ["precursor_mz", "precursormz", "precursor_mass"]
+_default_key = "precursor_mz"
+_accepted_keys = ["precursormz", "precursor_mass"]
```

Comment:
So this exists in parallel with the metadata harmonization, right?

Reply:
Yes, this is in principle a duplicate now. I would say it could go now, or what do you think?

Reply:
since `scan_start_time` is not among the harmonized keywords I'm leaving it here for now